### DISCUSSION TITLE
Contribua com esta organização!!

### DISCUSSION THREAD

**Discussion by OgliariNatan** [ENGENHARIA-DE-SOFTWARE-UNOPAR/.github#1] [General]
Caso curse algum curso de tecnologia na UNOPAR, contribua com esta repositório. <3.

**Comment by engleopsouza** [comment]
Opa, ola meu querido, eu tenho o drive com alguns materiais, voce tem material do 6o semestre de eng. soft.?!?

**Reply by OgliariNatan** [reply]
Olá, não tenho, ainda não cheguei lá, irei lhe adicionar nesta organização. 
Já deu uma olhada no [Modelo em latex para relatório](https://github.com/ENGENHARIA-DE-SOFTWARE-UNOPAR/Template-UNOPAR) ?

**Reply by engleopsouza** [reply]
Bacana, vou dar uma olhada neste modelo...
